controllers/apischeme/apischeme_controller.go

```diff
@@ -384,3 +363,10 @@ func sliceEquals(left, right []string) bool {
 	}
 	return true
 }
+
+// SetupWithManager sets up the controller with the Manager.
+func (r *APISchemeReconciler) SetupWithManager(mgr ctrl.Manager) error {
+	return ctrl.NewControllerManagedBy(mgr).
+		Owns(&cloudingressv1alpha1.APIScheme{}).
```

Comment:
Are we sure this is the right one, and not for example `For`? In the original controller it uses `EnqueueRequestForObject` rather than `EnqueueRequestForOwner` and CIO does not set an owner reference on apischemes.

Reply:
Yeah, I think it should be `For` instead of `Owns`. 

Fixed, thanks!